src/cosalib/cmdlib.py

```diff
@@ -447,9 +447,6 @@ def generate_image_json(srcfile):
     r = yaml.safe_load(open("/usr/lib/coreos-assembler/image-default.yaml"))
     for k, v in flatten_image_yaml(srcfile).items():
         r[k] = v
-    # Serialize our default GRUB config
-    with open("/usr/lib/coreos-assembler/grub.cfg") as f:
-        r['grub-script'] = f.read()
```

Comment:
ok, just to confirm 100% - this definitely isn't used anywhere?

Reply:
I could not find any other place using it, looking at https://github.com/coreos/coreos-assembler/commit/c9036faecbf9659809382c449a39c9e024fc7c3e it was needed for create_disk, what makes sense. So, I will say no